src/index.html

```diff
@@ -11,7 +11,214 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="formfield">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              required
+              name="surname"
+              id="surname"
+            >
+          </label>
+
+        </div>
```

Comment:
Add blank lines between child elements but don't add between parent and child
```suggestion
          </label>
        </div>
```